travis/large-requirements.txt

```diff
@@ -35,6 +35,7 @@ pytest-localserver==0.5.0
 sqlalchemy==1.3.0
 kubernetes==9.0.0
 pystan==2.17.1.0
-fbprophet>=0.5
+# TODO (harupy): Unpin fbprophet once https://github.com/facebook/prophet/issues/1339 is fixed.
```

Comment:
```suggestion
```

We actually want to keep `fbprophet` pinned, so we don't need to unpin it in the future.